dewy-client/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "dewy-client"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
Had to update this and run `poetry install` in `dewy client` in order for tests to see the new API. Once this goes in I'll plan to do a 0.1.2 release (or perhaps @kerinin should try to verify the release process)?